### PR TITLE
40 tk about us design

### PR DESCRIPTION
**Pull request by TobiasKjernell**

### CODE REVIEW

File: styles/aboutus.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+.team-content__member-fname,
+.team-content__member-lname {
+    font-family: 'Love Ya Like A Sister', sans-serif;
+    font-size: var(--large-font);
+    color: var(--textcolor-main);
+    padding-right: 3rem;
+    margin: 0;
+    text-align: right;
+}
```
Comment (jasminedns) [inline styles/aboutus.css:107]: You could add here to transform the names in uppercase

**Review by jasminedns — CHANGES_REQUESTED** [review]
Just a small thing that can be added later with JS, but why not put it now so we're fine with it.

File: styles/aboutus.css (hunk 1 above)
Comment (TobiasKjernell) [reply styles/aboutus.css:107]: ah yes that's true!

**Review by jasminedns — APPROVED** [review]
good to go!

**Review by Moon2024-light — APPROVED** [review]
looking good!

**Pull request merged**